### PR TITLE
 Accept npm module version

### PR DESCRIPTION
**Pull request by shanavas786**

### CODE REVIEW

**Comment by shanavas786** [comment]
Closes #58

**Comment by stultus** [comment]
@shanavas786 , Please let me know what is pending in this.

**Comment by shanavas786** [comment]
@stultus we were waiting for another collaborator to approve this PT.

**Comment by shanavas786** [comment]
@LeoIannacone @psuhailp 

Can you please have a look at this ?

**Comment by stultus** [comment]
We tested this PR the other day. And I can confirm that it is working as expected

**Comment by shanavas786** [comment]
> We tested this PR the other day. And I can confirm that it is working as expected

praveen also tested this before. But I cannot merge until approved by one of reviewers with write access

**Comment by shanavas786** [comment]
@LeoIannacone can you please add @stultus to collaborators ?

File: npm2deb/utils.py [hunk 1]
```diff
@@ -136,6 +136,10 @@ def create_dir(dir):
     except OSError as oserror:
         raise OSError("Error: directory %s already exists." % oserror.filename)
 
+def parse_name(name):
+    parts = name.partition('@')
```
Comment (LeoIannacone) [inline npm2deb/utils.py]: Are you sure that all the modules have this format ?

Comment (shanavas786) [reply npm2deb/utils.py]: we are parsing user input here.
if `@version` is not provided, npm defaults to latest version

**Pull request merged**